src/mongo/db/query/explain.cpp

```diff
@@ -62,6 +62,7 @@ using std::string;
 using std::unique_ptr;
 using std::vector;
 
+bool backupPlanUsed = false;
```

Comment:
This should be passed as a parameter to the function. These helpers can be called by multiple threads, which would induce a race condition writing to the variable here.